_posts/en/newsletters/2020-04-22-newsletter.md

```diff
@@ -0,0 +1,185 @@
+
+    This week, BTCPay [announced][btcpay pj announce] the release of
+    version 1.0.4.0 which includes an implementation of payjoin support
+    for both receiving payments in payment processor mode and sending
+    them using BTCPay's internal wallet.  For details on using the
+    protocol, see their [user guide][btcpay pj ug].  For technical
+    details on their implementation, see their [specification][btcpay pj
+    spec] or the [issue][btcpay-doc 486] where future improvements are
+    being actively discussed.  For this change to have the maximum
+    impact, other popular wallets need to implement support for creating
+    compatible payjoin payments.
+
+- **Lightning Labs drafts Lightning Service Authentication Tokens (LSAT) specification:**
+  Lightning Labs has [announced][ll lsat announcement] LSAT, a
+  [specification][lsat spec], which outlines a protocol for obtaining tokens
+  (macaroons) over LN and using them in an application as both authentication
+  and API payment mechanisms.
+
+- **Lightning Labs announces Faraday for channel management:**
+  [Faraday][ll faraday announcement] is a tool for LND node operators that
+  analyzes existing channels and makes recommendations to close bad,
+  under-performing channels. Such channels display attributes like low volume,
+  low uptime, or high fees.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding update to latest RCs/releases Tuesday afternoon -->
+
+- [Bitcoin Core 0.20.0rc1][bitcoin core 0.20.0] is a release candidate
+  for the next major version of Bitcoin Core.
+
+- [LND 0.10.0-beta.rc2][lnd 0.10.0-beta] allows testing the next major
+  version of LND.  Prospective testers are encouraged to read LND
+  developer Olaoluwa Osuntokun's [introduction to the release
+  candidate][lnd rc intro] from the [LND engineering mailing list][].
+
+- [C-Lightning 0.8.2-rc1][c-lightning 0.8.2] is the first release
+  candidate for the next version of C-Lightning.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Rust-Lightning][rust-lightning repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #17595][] adds support for [reproducibly building][topic
+  reproducible builds] Windows versions of Bitcoin Core using [GNU Guix][]
+  (pronounced "geeks"). The last remaining target platform for Guix reproducible
+  builds, macOS, has an open [draft PR][Bitcoin Core #17920].
+
+- [C-Lightning #3611][] adds a `keysend` plugin that allows a node to
+  securely receive [spontaneous payments][topic spontaneous
+  payments]---payments not preceded by the generation of an invoice.
```

Comment:
I think "payments not preceded by the generation of an invoice" is unnecessary since we're linking to the topic page.

Reply:
It might be a helpful description for those who don't click through.

Reply:
> It might be a helpful description for those who don't click through.

Indeed it is. My point is that we don't need to keep on defining concepts/terms that we've covered multiple times before and have a topic page for. *Spontaneous payments* have been defined in two previous newsletters (although the most recent was in June 2019).

It's subjective. I'd lean towards defining things fewer times, especially when we're linking to topics pages. Curious readers can easily click through for more details.